python/lsst/ctrl/execute/slurmPlugin.py

```diff
@@ -99,14 +99,8 @@ def countRunningSlurmJobs(jobname):
         numberOfJobs = SlurmPlugin.countSlurmJobs(jobname, jobstates="R")
         return numberOfJobs
 
-    def submit(self, platform, platformPkgDir):
-        configName = os.path.join(platformPkgDir, "etc", "config", "slurmConfig.py")
-
-        self.loadSlurm(configName, platformPkgDir)
-        verbose = self.isVerbose()
-        auto = self.isAuto()
+    def createFilesFromTemplates(self, platformPkgDir):
```

Comment:
Missing docstring

Reply:
Added a docsting with Parameters and Returns.